.eslintrc.json

```diff
@@ -4,6 +4,7 @@
         "node": true
     },
     "extends": "eslint:recommended",
+    "parser": "babel-eslint",
```

Comment:
Modifications to the project wide `.eslintrc.json` file will affect server side code too. We should not be applying linting rules to the server that are reflective of client side concerns, such as `parserOptions.sourceType = module`, which the server does not support.